### PR TITLE
Microprofile 5.0 Pre-Release 

### PR DESCRIPTION
**Pull request by emecas**
**Changes:**

      - Update README.adoc
      - Update architecture.asciidoc
      - Update required-apis.asciidoc

**Checklist:** https://wiki.eclipse.org/MicroProfile/SpecRelease/Release

1. Spec release creation

    Verify a spec release has been created for each new or update spec.
    For missing spec releases, notify spec teams by logging a github issue on that project and, if time-sensitive, contacting directly.

2. In MicroProfile project source

    NOTE: The build process will update the pom.xml <version>X.Y-SNAPSHOT</version> tags automatically, so do not manually change these.
 - [x]  Verify pom.xml has been updated with new spec dependencies (when a new specification is added). Spec teams should provide this, but worth the double-check.
 - [x]     Verify pom.xml has updated spec version numbers in the properties section. Spec teams should update this, but worth the double-check.
 - [x]     Verify architecture.adoc is updated with new specs and updated version numbers
 - [x]     Verify required-apis.adoc is updated with new specs and updated version numbers
 - [x]     Verify README.adoc is up-to-date with example POM file version number and any other spec changes.
 - [x]     Verify copyright dates have been updated in all changed files that mention these.
 - [ ]     Verify spec release pages contain a release text including incompatible changes and feature changes, linking to the "Changelog" section at a minimum.

3. Test build locally 
 - [x]      MicroProfile Clone the project and build locally before executing a release (mvn clean verify)
 - [x]     Look at log output. Even without errors, there may be warnings (improper javadoc annotations, etc). Notify spec teams by logging a GitHub issue on that project and, if time-sensitive, contacting directly. A RC (release candidate) will catch these early so they will not be as time-sensitive.
 - [x]     Verify proper MP spec PDF and HTML have been generated.

### CODE REVIEW

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 1]
```diff
@@ -32,6 +32,55 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile5.0-RC1]]
+=== MicroProfile 5.0 Release Candidate 1 (Q32021)
```
Comment (ederks85) [inline spec/src/main/asciidoc/architecture.asciidoc:36]: From MP 4.1 on, we decided to drop the quarter notion in this file. So this Q32021 notion can be removed

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 2]
```diff
@@ -32,6 +32,55 @@ Only this Specification will be generated and maintained to document the content
...
+// MicroProfile 5.0-RC1 is the 16th platform release for the
```
Comment (ederks85) [inline spec/src/main/asciidoc/architecture.asciidoc:37]: In 4.1, we decided to drop the number of platform releases we have done. So this "16th" notion can be removed

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 3]
```diff
@@ -32,6 +32,55 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile5.0-RC1]]
```
Comment (ederks85) [inline spec/src/main/asciidoc/architecture.asciidoc:35]: Per the following comments, I think it's best to have a look at the definition of the 4.1 release in this file. It has been updated a lot compared to 4.0.

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 4]
```diff
@@ -32,6 +32,55 @@ Only this Specification will be generated and maintained to document the content
...
+* https://download.eclipse.org/microprofile/staging/microprofile-open-api-3.0-RC1/microprofile-openapi-spec-3.0-RC1.html#_incompatible_changes[Open API]
+* https://download.eclipse.org/microprofile/microprofile-rest-client-3.0-RC2/microprofile-rest-client-spec-3.0-RC2.html[Rest Client - BROKEN LINK]
+* https://github.com/eclipse/microprofile-jwt-auth/releases/tag/2.0-RC2[JWT TODO]
+* https://download.eclipse.org/microprofile/staging/microprofile-opentracing-3.0-RC2/microprofile-opentracing-spec-3.0-RC2.html#_incompatible_changes[Open Tracing]
+
+If you are still dependent on Java EE 8, please consider using the https://github.com/eclipse/microprofile/releases/tag/4.1[4.1 release of MicroProfile].
+If you are still dependent on Java EE 7, please consider using the https://github.com/eclipse/microprofile/releases/tag/1.4[1.4 release of MicroProfile].
+
+The complete list of MicroProfile 5.0-RC1 specifications includes:
+
```
Comment (ederks85) [inline spec/src/main/asciidoc/architecture.asciidoc:59]: The versions between brackets need to match with the links

File: spec/src/main/asciidoc/required-apis.asciidoc [hunk 5]
```diff
@@ -23,19 +23,19 @@
...
- - <<jakarta-annotations, Jakarta Annotations 1.3>>
- - <<mp-config, MicroProfile Config 2.0>>
- - <<mp-fault-tolerance, MicroProfile Fault Tolerance 3.0>>
- - <<mp-health-check, MicroProfile Health 3.1>>
- - <<mp-jwt-auth, MicroProfile JWT Authentication 1.2>>
- - <<mp-metrics, MicroProfile Metrics 3.0>>
- - <<mp-open-api, MicroProfile OpenAPI 2.0>>
- - <<mp-opentracing, MicroProfile OpenTracing 2.0>>
- - <<mp-rest-client, MicroProfile Rest Client 2.0>>
+ - <<jakarta-cdi, CDI (Jakarta Contexts and Dependency Injection) 2.0>> ?
```
Comment (ederks85) [inline spec/src/main/asciidoc/required-apis.asciidoc:26]: You need to look up the exact versions of the Jakarta specifications used in Jakarta EE 9.1

File: spec/src/main/asciidoc/required-apis.asciidoc [hunk 6]
```diff
@@ -98,70 +98,70 @@ Use of implementations beyond Jakarta Annotations 1.3 in MicroProfile is allowed
  - https://github.com/eclipse-ee4j/common-annotations-api
 
 [[mp-config]]
-=== MicroProfile Config 2.0
+=== MicroProfile Config 3.0-RC4
```
Comment (ederks85) [inline spec/src/main/asciidoc/required-apis.asciidoc:101]: Before merging the PR, please look up if the specs have not released any now RC's yet

**Comment by ederks85** [comment]
Good progress @emecas. When the PR is complete, please use a squash merge since it's not really necessary to keep the independent commits in this context. And is much cleaner in the history :)

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 2 above)
Comment (emecas) [reply spec/src/main/asciidoc/architecture.asciidoc:37]: line is commented, anyway I'm removing it!!

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 1 above)
Comment (emecas) [reply spec/src/main/asciidoc/architecture.asciidoc:36]: Done

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 4 above)
Comment (emecas) [reply spec/src/main/asciidoc/architecture.asciidoc:58]: It matches with the link, does not it? are you referring to the line [56] recommending MP1.4 if you still using Java EE 7?

File: spec/src/main/asciidoc/required-apis.asciidoc (hunk 5 above)
Comment (emecas) [reply spec/src/main/asciidoc/required-apis.asciidoc:26]: Do you know where I can look for that ? I guess by observing the Jakarta EE 9.1 dependencies in the POM, right?

Comment (emecas) [reply spec/src/main/asciidoc/required-apis.asciidoc]: Updated!!

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 4 above)
Comment (ederks85) [reply spec/src/main/asciidoc/architecture.asciidoc:58]: I think these above are fine, but these below are not aligned yet. E.g.: /tag/3.0-RC4[MicroProfile Config 2.0]

Comment (emecas) [reply spec/src/main/asciidoc/architecture.asciidoc:58]: Ohhhhh I see what you meant, these are fixed now!!! thanks

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 3 above)
Comment (emecas) [reply spec/src/main/asciidoc/architecture.asciidoc:35]: I think in addition to your recommendation about having a look at 4.1 definition, maybe this link could summarize the most important about  5.0: 

https://projects.eclipse.org/projects/technology.microprofile/releases/5.0

what do you think ?

Comment (ederks85) [reply spec/src/main/asciidoc/architecture.asciidoc:35]: Correct, this is the link to the release plan that should contain all necessary details of the release

**Comment by ederks85** [comment]
It's starting to look great @emecas. I suggest you tick off all the boxes, marking the PR as complete as it gets. Then we can get an RC-1 out and people can start experimenting with MP 5.0. During that time, we can update links to the specifications when they come.

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 7]
```diff
@@ -32,6 +32,59 @@ Only this Specification will be generated and maintained to document the content
...
+    Health 4.0
+    JWT RBAC 2.0
+    Metrics 4.0
+    Open API 3.0
+    Open Tracing 3.0
+    Rest Client 3.0
+
+The following specifications include API incompatible changes:
+
+* https://download.eclipse.org/microprofile/staging/microprofile-config-3.0-RC4/microprofile-config-spec-3.0-RC4.html#_incompatible_changes[Config]
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/architecture.asciidoc:51]: please use 3.0-RC5

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 8]
```diff
@@ -32,6 +32,59 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile5.0-RC1]]
+=== MicroProfile 5.0 Release Candidate 1
+
+MicroProfile 5.0-RC1 is aligning every specification with Jakarta EE 9.1. This effectively creates an incompatible change due to the namespace change from javax to jakarta in the code.
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/architecture.asciidoc:38]:
```suggestion
MicroProfile 5.0-RC1 aligns with Jakarta EE 9.1. It means this release will not work with previous version of Jakarta EE such as Jakarta EE 8. This effectively creates an incompatible change due to the namespace change from javax to jakarta in the code. 
```

File: spec/src/main/asciidoc/required-apis.asciidoc (hunk 6 above)
Comment (Emily-Jiang) [reply spec/src/main/asciidoc/required-apis.asciidoc:101]: Please use 3.0-RC5

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 8 above)
Comment (emecas) [reply spec/src/main/asciidoc/architecture.asciidoc:38]: Done!! Thanks

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 9]
```diff
@@ -32,6 +32,59 @@ Only this Specification will be generated and maintained to document the content
...
+    JWT RBAC 2.0
+    Metrics 4.0
+    Open API 3.0
+    Open Tracing 3.0
+    Rest Client 3.0
+
+The following specifications include API incompatible changes:
+
+* https://download.eclipse.org/microprofile/staging/microprofile-config-3.0-RC5/microprofile-config-spec-3.0-RC5.html#_incompatible_changes[Config]
+* https://download.eclipse.org/microprofile/staging/microprofile-fault-tolerance-4.0-RC2/microprofile-fault-tolerance-spec-4.0-RC2.html#_incompatible_changes[Fault Tolerance]
```
Comment (xstefank) [inline spec/src/main/asciidoc/architecture.asciidoc:52]: these should be moved out of /staging directory before we push 5.0-RC1

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 10]
```diff
@@ -32,6 +32,59 @@ Only this Specification will be generated and maintained to document the content
...
+* https://github.com/eclipse/microprofile-health/releases/tag/4.0-RC2[Health TODO]
```
[the remark this answers is not in the document]
Comment (emecas) [reply spec/src/main/asciidoc/architecture.asciidoc]: Updated!!

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 9 above)
Comment (ederks85) [reply spec/src/main/asciidoc/architecture.asciidoc:52]: @emecas let's monitor these staging links. When the links without the "staging" part is available, the links can be updated here

Comment (Emily-Jiang) [reply spec/src/main/asciidoc/architecture.asciidoc:52]: I'll release more update for the specs in the next couple of days and will push the latest out of staging.

File: spec/src/main/asciidoc/required-apis.asciidoc [hunk 11]
```diff
@@ -98,70 +98,70 @@ Use of implementations beyond Jakarta Annotations 1.3 in MicroProfile is allowed
...
 
 [[mp-config]]
-=== MicroProfile Config 2.0
+=== MicroProfile Config 3.0-RC5
 
 MicroProfile Config provides applications and microservices means to obtain configuration properties through several environment-aware sources both internal and external to the application and made available through dependency injection or lookup.
 
  - http://microprofile.io/project/eclipse/microprofile-config
- - https://github.com/eclipse/microprofile-config/releases/tag/2.0
+ - https://github.com/eclipse/microprofile-config/releases/tag/3.0-RC4
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/required-apis.asciidoc:106]:
```suggestion
 - https://github.com/eclipse/microprofile-config/releases/tag/3.0-RC5
```

**Review by ederks85 — APPROVED** [review]
Aside from some links or specification RC versions, I think it's fine to merge this and create a MP 5.0-RC1 release so that implementations can start experimenting/adopting.

@Emily-Jiang @radcortez do you concur?

**Review by xstefank — COMMENTED** [review]
@ederks85 Health is already at RC3 but with only license changes so no pressure to change that. Aside from that I still think that staging links can be moved out of staging and updated even for RC. Let me know if you'd like to help with that. Also, there are still some TODOs and similar (Metrics TODO, Rest Client - BROKEN LINK). Do we want to release these?

**Comment by ederks85** [comment]
@xstefank releasing an RC should only happen when there are no trivial issues open, so if we need to wait until the TODOs and links are fixed, I'm all for it. I can push certain staging buttons if that speeds up the process, but I need someone to tell me how and when since I'm not involved in these developments

**Comment by xstefank** [comment]
@ederks85 maybe that's just how I understand RC (release candidates). It's potentially a final release so it should have everything in place as final release. But I might be wrong. You don't need to block the PR on my comments.

**Comment by emecas** [comment]
> @ederks85 maybe that's just how I understand RC (release candidates). It's potentially a final release so it should have everything in place as final release. But I might be wrong. You don't need to block the PR on my comments.

@xstefank  thanks for your feedback, I will take care of all those minimum details in order to complete documentation as closest as possible to the final release

@ederks85 let me do another round of refining

**Comment by starksm64** [comment]
Since this is already a breaking change release, why are we not dropping open tracing? It is clear that Micrometer is the future of metrics.

**Comment by Emily-Jiang** [comment]
> Since this is already a breaking change release, why are we not dropping open tracing? It is clear that Micrometer is the future of metrics.

The only breaking changes is namespace change in MP 5.0. No other changes are introduced so that we can help developers limit the migration effort. As for MP Metrics, I am afraid it needs more conversations. MP might continue with Dropwizard if customers are still happy. In the meantime, more effort is spending on investigating micrometer and OpenTelemetry. Before we have an alternative, I don’t think it’s a good move to remove technologies from the release.

**Comment by Emily-Jiang** [comment]
@ederks85 @emecas I have done some fix and released all of MP component specs. The latest release version is [here](https://docs.google.com/spreadsheets/d/1yYI5pdRCoOT5GaZ6KMNTfg3mQ5Hwxxv45SW5WBsZAmY/edit#gid=29484381). To @xstefank 's point, I have done the post release step and all of the release candidate should end up in the download dir instead of staging. Once you sync up that, I think this PR is good to go.

**Comment by emecas** [comment]
> 
> 
> @ederks85 @emecas I have done some fix and released all of MP component specs. The latest release version is [here](https://docs.google.com/spreadsheets/d/1yYI5pdRCoOT5GaZ6KMNTfg3mQ5Hwxxv45SW5WBsZAmY/edit#gid=29484381). To @xstefank 's point, I have done the post release step and all of the release candidate should end up in the download dir instead of staging. Once you sync up that, I think this PR is good to go.

@Emily-Jiang  Updated, but I found 2 things :

- A broken link in Config to Spec html (https://download.eclipse.org/microprofile/staging/microprofile-config-3.0-RC5/microprofile-config-spec-3.0-RC5.html):
https://download.eclipse.org/microprofile/staging/microprofile-config-3.0-RC5/microprofile-config-spec-3.0-RC5.html#_incompatible_changes

- For metrics this section  SHOULD NOT BE incompatible changes ?  https://download.eclipse.org/microprofile/microprofile-metrics-4.0-RC2/microprofile-metrics-spec-4.0-RC2.html#_breaking_changes

**Comment by Emily-Jiang** [comment]
> > @ederks85 @emecas I have done some fix and released all of MP component specs. The latest release version is [here](https://docs.google.com/spreadsheets/d/1yYI5pdRCoOT5GaZ6KMNTfg3mQ5Hwxxv45SW5WBsZAmY/edit#gid=29484381). To @xstefank 's point, I have done the post release step and all of the release candidate should end up in the download dir instead of staging. Once you sync up that, I think this PR is good to go.
> 
> @Emily-Jiang Updated, but I found 2 things :
> 
>     * A broken link in Config to Spec html (https://download.eclipse.org/microprofile/staging/microprofile-config-3.0-RC5/microprofile-config-spec-3.0-RC5.html):
>       https://download.eclipse.org/microprofile/staging/microprofile-config-3.0-RC5/microprofile-config-spec-3.0-RC5.html#_incompatible_changes
> 
I moved the artificates for the permanent location based on Martin's comments but I forgot to update the link on the release page. These links are now updated.
>     * For metrics this section  SHOULD NOT BE incompatible changes ?  https://download.eclipse.org/microprofile/microprofile-metrics-4.0-RC2/microprofile-metrics-spec-4.0-RC2.html#_breaking_changes

Updating namespace from javax to jakarta is a break change.

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 12]
```diff
@@ -32,6 +32,59 @@ Only this Specification will be generated and maintained to document the content
...
+    Open Tracing 3.0
+    Rest Client 3.0
+
+The following specifications include API incompatible changes:
+
+* https://download.eclipse.org/microprofile/staging/microprofile-config-3.0-RC5/microprofile-config-spec-3.0-RC5.html#_incompatible_changes[Config --> BROKEN]
+* https://download.eclipse.org/microprofile/microprofile-fault-tolerance-4.0-RC3/microprofile-fault-tolerance-spec-4.0-RC3.html#_incompatible_changes[Fault Tolerance]
+* https://download.eclipse.org/microprofile/microprofile-health-4.0-RC3/microprofile-health-spec-4.0-RC3.html#_incompatible_changes[Health]
+* https://download.eclipse.org/microprofile/microprofile-metrics-4.0-RC2/microprofile-metrics-spec-4.0-RC2.html#_breaking_changes[Metrics --> This section SHOULD BE incompatible changes ]
+* https://download.eclipse.org/microprofile/microprofile-open-api-3.0-RC2/microprofile-openapi-spec-3.0-RC2.html#_incompatible_changes[Open API]
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/architecture.asciidoc:55]:
```suggestion
* https://download.eclipse.org/microprofile/microprofile-open-api-3.0-RC3/microprofile-openapi-spec-3.0-RC3.html#_incompatible_changes[Open API]
```

**Comment by Emily-Jiang** [comment]
> Ah, I realised what you meanted. Let me fix the wording and do a further release on metrics spec.

I have fixed it under this [PR](https://github.com/eclipse/microprofile-metrics/pull/645). Please review @ederks85

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 12 above)
Comment (emecas) [reply spec/src/main/asciidoc/architecture.asciidoc]: Thanks, I've updated  open-api to RC3 in 3 more places

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 13]
```diff
@@ -32,6 +32,59 @@ Only this Specification will be generated and maintained to document the content
...
+    Health 4.0
+    JWT RBAC 2.0
+    Metrics 4.0
+    Open API 3.0
+    Open Tracing 3.0
+    Rest Client 3.0
+
+The following specifications include API incompatible changes:
+
+* https://download.eclipse.org/microprofile/staging/microprofile-config-3.0-RC5/microprofile-config-spec-3.0-RC5.html#_incompatible_changes[Config]
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/architecture.asciidoc:51]:
```suggestion
* https://download.eclipse.org/microprofile/microprofile-config-3.0-RC5/microprofile-config-spec-3.0-RC5.html#_incompatible_changes[Config]
```

**Comment by Emily-Jiang** [comment]
> > > Ah, I realised what you meanted. Let me fix the wording and do a further release on metrics spec.
> > 
> > 
> > I have fixed it under this [PR](https://github.com/eclipse/microprofile-metrics/pull/645). Please review @ederks85
> 
> Great, thank you for fixing taht now all specs use `incompatible changes` wording:
> 
> Only 2 things in pending at this point:
> 
>     * [ ]  Metric spec page is not reflecting this change yet.
> 
>     * [x]  ~also Config  spec link is still broken.~

@emecas I have updated your link as you still point to the staging site.
I need to release a new verison of Metrics later today. I will make a update suggestion once the release is ready.

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 13 above)
Comment (emecas) [reply spec/src/main/asciidoc/architecture.asciidoc]: Fixed!!! thx

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 14]
```diff
@@ -32,6 +32,59 @@ Only this Specification will be generated and maintained to document the content
...
+    Open API 3.0
+    Open Tracing 3.0
+    Rest Client 3.0
+
+The following specifications include API incompatible changes:
+
+* https://download.eclipse.org/microprofile/microprofile-config-3.0-RC5/microprofile-config-spec-3.0-RC5.html#_incompatible_changes[Config]
+* https://download.eclipse.org/microprofile/microprofile-fault-tolerance-4.0-RC3/microprofile-fault-tolerance-spec-4.0-RC3.html#_incompatible_changes[Fault Tolerance]
+* https://download.eclipse.org/microprofile/microprofile-health-4.0-RC3/microprofile-health-spec-4.0-RC3.html#_incompatible_changes[Health]
+* https://download.eclipse.org/microprofile/microprofile-metrics-4.0-RC2/microprofile-metrics-spec-4.0-RC2.html#_breaking_changes[Metrics --> This section SHOULD BE incompatible changes ]
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/architecture.asciidoc:54]:
```suggestion
* https://download.eclipse.org/microprofile/microprofile-metrics-4.0-RC4/microprofile-metrics-spec-4.0-RC4.html#_incompatible_changes[Metrics]
```

File: spec/src/main/asciidoc/architecture.asciidoc [hunk 15]
```diff
@@ -32,6 +32,59 @@ Only this Specification will be generated and maintained to document the content
...
+    JWT RBAC 2.0
+    Metrics 4.0
+    Open API 3.0
+    Open Tracing 3.0
+    Rest Client 3.0
+
+The following specifications include API incompatible changes:
+
+* https://download.eclipse.org/microprofile/microprofile-config-3.0-RC5/microprofile-config-spec-3.0-RC5.html#_incompatible_changes[Config]
+* https://download.eclipse.org/microprofile/microprofile-fault-tolerance-4.0-RC3/microprofile-fault-tolerance-spec-4.0-RC3.html#_incompatible_changes[Fault Tolerance]
```
Comment (Emily-Jiang) [inline spec/src/main/asciidoc/architecture.asciidoc:52]:
```suggestion
* https://download.eclipse.org/microprofile/microprofile-fault-tolerance-4.0-RC4/microprofile-fault-tolerance-spec-4.0-RC4.html#_incompatible_changes[Fault Tolerance]
```

**Comment by Emily-Jiang** [comment]
@emecas @ederks85 I have done all of the fixes, I believe. If you take in my suggestions, it will be good to go.

**Review by Emily-Jiang — APPROVED** [review]
I have added a couple of suggestions. Please check and then merge in my suggestions.

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 14 above)
Comment (emecas) [reply spec/src/main/asciidoc/architecture.asciidoc]: Fixed

File: spec/src/main/asciidoc/architecture.asciidoc (hunk 15 above)
Comment (emecas) [reply spec/src/main/asciidoc/architecture.asciidoc]: Fixed

**Comment by emecas** [comment]
> 
> 
> I have added a couple of suggestions. Please check and then merge in my suggestions.

@ederks85 I had applied all @Emily-Jiang  suggestions. This PR is ready to be merged, thanks to everybody for contributing!!

**Comment by emecas** [comment]
> 
> 
> I have added a couple of suggestions. Please check and then merge in my suggestions.

@ederks85 I had applied all @Emily-Jiang suggestions. This PR is ready to be merged, thanks to everybody for contributing!!

**Comment by ederks85** [comment]
Great @emecas , I will merge the PR and get back to you for continuing the release

**Pull request merged**